### ISSUE TITLE
Minimum Dataset Count

### ISSUE THREAD

**Issue by contactvpatel** [githubharald/SimpleHTR#51]
* TensorFlow version 1.12
* Python version 3.5
* Operating system - Windows 10

I am using 2,823 images to train a model with numTrainSamplesPerEpoch as 25000 and batchSize as 100. I am not getting accuracy more than 14%. I have tried playing around those values (numTrainSamplesPerEpoch and batchSize) but don't see much improvement.

Is my dataset (2,823 images) are good enough for training a model? What would be minimum dataset size to train a model?

My sample project is attached. Please take a look and let me know how I can improve accuracy for a model.

[SimpleHTRTest.zip](https://github.com/githubharald/SimpleHTR/files/2886273/SimpleHTRTest.zip)

**Comment by githubharald** [comment]
this is not an issue with the code, therefore closing it.
